packages/fluentui/react-northstar/test/specs/components/RadioGroup/RadioGroup-test.tsx

```diff
@@ -81,6 +82,29 @@ describe('RadioGroup', () => {
       expect(onClick).toHaveBeenCalled();
     });
 
+    it('calls onChange handler for item with updated checked state', () => {
+      const items = getItems();
+      const wrapper = mountWithProvider(<RadioGroup items={items} />);
+      const radioGroupItems = wrapper.find('RadioGroupItem');
+
+      radioGroupItems
+        .first()
+        .find('div')
+        .first()
+        .simulate('click');
+      wrapper.update();
+      const onChange = items[0].onChange || items[0].props.onChange;
+      expect(onChange).toHaveBeenCalledWith(undefined, expect.objectContaining({ checked: true }));
+
+      radioGroupItems
+        .last()
+        .find('div')
+        .first()
+        .simulate('click');
+      wrapper.update();
```

Comment:
Should be `act()` used instead?

Reply:
is it needed ? enzyme should wrap all actions with `act`

Reply:
might as well try removing the `wrapper.update()`

Reply:
Removing it will not call `onChange`